discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -90,6 +90,8 @@ def _terraform_params(self, **kwargs):
                 kwargs.get("master_disk_count", 1), consts.OperatorResource.MASTER_DISK_COUNT_KEY, operators),
             "worker_disk_count": utils.resource_param(
                 kwargs.get("worker_disk_count", 1), consts.OperatorResource.WORKER_DISK_COUNT_KEY, operators),
+            "worker_cpu_mode": kwargs.get("worker_cpu_mode", consts.VM_CPU_MODE),
+            "master_cpu_mode": kwargs.get("master_cpu_mode", consts.VM_CPU_MODE)
```

Comment:
Why not to use 2 variables. one for worker and the other for master?

Reply:
i assumed that this will never used except of very specific flows, do you think there is a need to have 2 different?

Reply:
ATM it is only this use case but in the future someone could change it. For clarity I suggest to use 2.

Reply:
sure, done